conversion/public/bite.v1.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+    self = this;
+
+    var actions = {
+        'init': function (programId) {
+            self.programId = programId;
+        },
+        'send': function (hitType, eventCategory, eventAction, eventLabel, eventValue) {
+            if (!self.programId) {
+                throw new Error('BarnebysAnalytics is not initialized correctly');
+            }
+
+            var query = '/?p=' + self.programId +
+                '&k='  + hitType +
+                '&d1=' + eventCategory +
+                '&d2=' + eventAction +
+                '&d3=' + eventLabel +
+                '&d4=' + eventValue;
+
+            var request = new XMLHttpRequest();
+            request.open('GET', 'http://localhost:3000' + query);
+            request.send();
+        }
+    };
+
+    function performAction(actionName, ...args) {
+        var action = actions[actionName] || (function () { console.log(action, 'not implemented') });
+        action(...args);
+    };
+
+    var actionQueue = window[window['BarnebysAnalyticsObject']].q;
+    actionQueue.forEach(function ([action, ...args]) {
+        performAction(action, ...args);
```

Comment:
Spread operators will not work with older browsers

Reply:
We should probably use webpack (or similar) to compile and bundle the client script later on. Will accept this for now.

Reply:
Yes, up until yesterday I wasn’t sure about how it would be deployed and wanted to keep build steps down to a minimum.

However, aince we decided yesterday that W e’ll use now for this, it should be relatively easy to add webpack or similar preprocessing